### ISSUE TITLE
Bluespace Polycrystal Sheets Can't Be Used For Machine Construction (+ Related)

### ISSUE THREAD

**Issue by AnonymousNow** [tgstation/tgstation#47650]
Bluespace polycrystals are the smelted "sheet" form of bluespace crystals that you can get from the ORM and furnaces. However, despite ostensibly being made out of a number of individual usable crystals (you can snap them off in your hands), they're not counted as bluespace crystals for the purposes of adding them to machinery (eg. teleporters), and likely for other purposes, too. You have to snap them off individually first.

This is minor, but it's a consistency issue, and it bugs me. To that end, a single sheet of bluespace polycrystal is still counted as such, despite literally being a crystal - while it makes sense from a gameplay perspective, as you effectively destroy the polycrystal sheets when snapping the last one out of its state and therefore it's difficult to make more polycrystal (which you can add polycrystal to, like adding stacks of other materials to themselves) without going to the ORM, it makes little sense from a logical perspective, being "this one of something is a singular unit of its multiple form", and that offends my autism.

There seems to be some sort of marker as to which crystals have already been put through the ORM, likely resulting from them being sourced from polycrystal and required to prevent infinite crystal production from looping them through the ORM. Therefore, I have a few proposals:

- To fix the main problem, just make polycrystals usable in that form for machinery and other things which accept bluespace crystals. Slap a couple of sheets into the teleporter machines rather than snapping them off individually. Easy, done.
- To fix the other problem, have a singular polycrystal not be a polycrystal, but instead just a bluespace crystal. This would still have the marker of being smelted. Using crystals with this marker on each other makes polycrystals again.
- To more clearly delineate which crystals have been put through the ORM, and therefore can't be put through again or they'll be destroyed (and also can be added together, as in my above suggestion), I propose that singular instances of bluespace polycrystal should be renamed "stabilised bluespace crystals" instead, indicating that they shouldn't be resmelted or they'll be gone. Could even have a differently shaped sprite, if you're inclined.

tl;dr - bluespace polycrystals offend my autism, pls fix.

**Comment by TheVekter** [comment]
Yeah, this has been a thing for a while. I noticed it while building one of the station goals a while back. I can't imagine this is a hard fix, but the materials datum vexes the fuck out of me.

**Comment by nemvar** [comment]
I'm not sure this is even material datum related.

**Comment by kriskog** [comment]
Nah this is years old
